components/SiteNavbar.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <nav class="flex flex-wrap flex-row-reverse justify-between border-b border-gray-300 px-5 py-3 lg:flex-row">
+    <div class="flex lg:hidden" @click="mobileView = !mobileView">
```

Comment:
It might just be me being a component fanatic, but this probably could be split in multiple smaller component to improve readability of the navbar? Again, not blocking for now

Reply:
Agreed that these should be split up now before it gets too verbose. I'll break these down into smaller components.